test/suite/wttest.py

```diff
@@ -623,7 +623,7 @@ def tearDown(self, dueToRetry=False):
         if passed and self.__module__.startswith("test_layered"):
             # FIXME-WT-15786: Handle the transient state where a follower that has not yet picked up
             # its first checkpoint may fail with ENOENT due to missing its stable table.
-            if not re.match("test_layered(57|41|21|22|17)", str(self)):
+            if not re.match("test_layered(61|57|41|22|21|17)", str(self)):
```

Comment:
@mariammojid FYI, looks like @quchenhao faced an issue and had to skip a test.

Reply:
@quchenhao can you please specify what error message you encounter with those tests in particular?

Reply:
```
[pid:9830]: Traceback (most recent call last):
  File "/home/ubuntu/wiredtiger/test/suite/wttest.py", line 627, in tearDown
    self.verifyLayered()
  File "/home/ubuntu/wiredtiger/test/suite/wttest.py", line 598, in verifyLayered
    self.verifyUntilSuccess(sess, uri)
  File "/home/ubuntu/wiredtiger/test/suite/wttest.py", line 929, in verifyUntilSuccess
    return self.retryEBUSY(session, lambda: session.verify(uri, config), **kwargs)
  File "/home/ubuntu/wiredtiger/test/suite/wttest.py", line 908, in retryEBUSY
    return func()
  File "/home/ubuntu/wiredtiger/test/suite/wttest.py", line 929, in <lambda>
    return self.retryEBUSY(session, lambda: session.verify(uri, config), **kwargs)
_wiredtiger.WiredTigerError: Device or resource busy
```

Reply:
Interesting. I checked out into your branch, removed them from the regex that was excluding them, and I passed `test_layered(21|22|61)`. That's very strange. @quchenhao can you please try merging develop into your branch and retry with a new build dir?

Reply:
yes, it is passing after I merged with develop. Thanks.